static/js/field_monitor_display.js

```diff
@@ -4,12 +4,20 @@
 // Client-side logic for the field monitor display.
 
 var websocket;
+var currentMatchId;
 var redSide;
 var blueSide;
 var lowBatteryThreshold = 8;
 
-// Handles a websocket message to update the team connection status.
+
 var handleArenaStatus = function(data) {
+  // If getting data for the wrong match (e.g. after a server restart), reload the page.
+  if (currentMatchId == null) {
+    currentMatchId = data.MatchId;
+  } else if (currentMatchId !== data.MatchId) {
+    location.reload();
+  }
```

Comment:
Is this not happening automatically? In theory, when the page loses its websocket connection to the server it should retry connecting every three seconds, and when it reconnects it should get all the events that populate the initial state.

Reply:
Not always after Cheesy is stopped and started.